### PR TITLE
feat(cat-voices): back buttons in get create recovery dialog

### LINKED ISSUE #1083: 🛠️ [TASK] : MVE2 refinement: change get started method
### Summary

Change get started method

### Description

Child of #727 

As a user, I want to be able to go back after selecting a get started method (create new vs recover), so that I can choose a different option with minimal clicks (in case I chose the wrong one by mistake).

AC:
- Add "Back" button on "Create your Catalyst Keychain" page
- Add "Back" button on "Restore your Catalyst Keychain" page

### PR DESCRIPTION
**Pull request by LynxLynxx**
# Description

Adding buttons on the "Create your Catalyst Keychain" page and "Restore your Catalyst Keychain" page.

## Related Issue(s)

Closes #1083 

## Description of Changes

Adding VoicesBackButton on the "Create your Catalyst Keychain" page and "Restore your Catalyst Keychain" page. This buttons should take you back to the screen when you can select method of getting started.

## Screenshots

image
image

## Please confirm the following checks

* [x] My code follows the style guidelines of this project
* [x] I have performed a self-review of my code
* [x] I have commented my code, particularly in hard-to-understand areas
* [x] I have made corresponding changes to the documentation
* [x] My changes generate no new warnings
* [x] I have added tests that prove my fix is effective or that my feature works
* [x] New and existing unit tests pass locally with my changes
* [x] Any dependent changes have been merged and published in downstream module

### CODE REVIEW

File: catalyst_voices/utilities/uikit_example/devtools_options.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+description: This file stores settings for Dart & Flutter DevTools.
+documentation: https://docs.flutter.dev/tools/devtools/extensions#configure-extension-enablement-states
```
Comment (dtscalac) [inline catalyst_voices/utilities/uikit_example/devtools_options.yaml:2]: Do we need this file?

Comment (LynxLynxx) [reply catalyst_voices/utilities/uikit_example/devtools_options.yaml:2]: We don't. My ide generated them automatically i remove them.

File: catalyst_voices/apps/voices/devtools_options.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+description: This file stores settings for Dart & Flutter DevTools.
```
[the remark this answers is not in the document]
Comment (LynxLynxx) [reply catalyst_voices/apps/voices/devtools_options.yaml:1]: We don't. My ide generated them automatically i remove them.

**Review by dtscalac — APPROVED** [review]
LGTM

**Pull request merged**